tools/depends/target/ffmpeg/CMakeLists.txt

```diff
@@ -58,11 +58,8 @@ elseif(CORE_SYSTEM_NAME STREQUAL osx)
                           --disable-decoder=mpeg_xvmc --disable-vda --disable-crystalhd --enable-videotoolbox
                           --target-os=darwin
                           --disable-securetransport)
-elseif(CORE_SYSTEM_NAME STREQUAL rbpi)
-  list(APPEND ffmpeg_conf --cpu=${CPU} --disable-vaapi --disable-vdpau)
```

Comment:
is FFmpeg still compiling on rbpi, because vaapi and vdpau are now enabled for it?
see https://github.com/xbmc/xbmc/commit/853c7f6403dfe29aec0b40d7d1c9ab787858daab for reference